### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by anderspapitto**

### CODE REVIEW

File: .jenkins/caffe2/build.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ if [[ "${BUILD_ENVIRONMENT}" == conda* ]]; then
   # headers are those in conda as well
   # This path comes from install_anaconda.sh which installs Anaconda into the
   # docker image
-  PROTOBUF_INCDIR=/opt/conda/include pip install "${ROOT_DIR}/third_party/onnx"
+  PROTOBUF_INCDIR=/opt/conda/include CMAKE_ARGS="${CMAKE_ARGS} -DONNX_NO_WERROR=ON" pip install "${ROOT_DIR}/third_party/onnx"
```
Comment (bddppq) [inline .jenkins/caffe2/build.sh]: Hmm this means we need to relax the checks in onnx, shouldn't ask users to pass additional flags to install onnx

File: cmake/Dependencies.cmake [hunk 2]
```diff
@@ -599,7 +599,13 @@ set(TEMP_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
 # We will build onnx as static libs and embed it directly into the binary.
 set(BUILD_SHARED_LIBS OFF)
 set(ONNX_USE_MSVC_STATIC_RUNTIME ${CAFFE2_USE_MSVC_STATIC_RUNTIME})
-set(ONNX_NO_WERROR ON)
+
+
+# See https://stackoverflow.com/questions/3766740/overriding-a-default-option-value-in-cmake-from-a-parent-cmakelists-txt
+set(ONNX_NO_WERROR ON CACHE BOOL "Disable use of Werror")
```
Comment (bddppq) [inline cmake/Dependencies.cmake]: yeah this is asking too much for projects that integrate with onnx (I mean they can easily fall into this cmake trap).

**Review by bddppq — APPROVED** [review]
Let's land this soon to fix the caffe2 CI, but please it's better to change onnx cmake files to relax these checks.

File: .jenkins/caffe2/build.sh (hunk 1 above)
Comment (anderspapitto) [reply .jenkins/caffe2/build.sh]: ++

**Pull request merged**